test/library/standard/FileSystem/chmod/chmod_noprivilege.chpl

```diff
@@ -0,0 +1,9 @@
+use FileSystem;
+// Test when the user tries to modify a file or a directory without privilege
+// Should throw PermissionError assuming user cannot modify root folder
+config const permissions = 0o777;
+config const original = 0o755;
```

Comment:
This assumes a specific setting for "/" that might not be true on all systems.  I think it would be better to use [this function](https://chapel-lang.org/docs/latest/modules/standard/FileSystem.html#FileSystem.getMode) to get the original permissions, and then determine a good modification candidate from that.

With that in mind, these don't need to be `config`s and the .execopts file can be removed